# message for multi uav communication
time timestamp
# UAV id
int8 uav_id
# descriptors data
sensor_msgs/Image descriptors

